### PR TITLE
chore: adjusted code by ryan-woodruff to use context of migration ins…

### PR DESCRIPTION
**Pull request by bielu**
Hello @KevinJump,
Had some spare time on my hands, so adjusted code started by @ryan-woodruff, to use context + be compatible with latest changes in core.
https://github.com/Jumoo/uSyncMigrations/pull/217
Sadly I dont have any grid based site with layouts to properly test it :/, Changes are basically same as in 217, but with removal of umbraco services and usage of context instead :)

### CODE REVIEW

**Comment by ryan-woodruff** [comment]
Hi ! - Thanks for picking this up, I meant to pick it up but didn't find time.

I can take a look at testing it at some point next week :)

File: uSync.Migrations/Migrators/BlockGrid/GridToBlockGridMigrator.cs [hunk 1]
```diff
@@ -112,6 +118,13 @@ private ILegacyGridEditorsConfig GetGridConfig(SyncMigrationContext context)
 			return string.Empty;
 		}
 
+		var dataTypeAlias = context.ContentTypes.GetDataTypeAlias(contentProperty.ContentTypeAlias, contentProperty.PropertyAlias);
```
Comment (ryan-woodruff) [inline uSync.Migrations/Migrators/BlockGrid/GridToBlockGridMigrator.cs:121]: Hi @bielu & @KevinJump , apologies for the delay in testing this.

I've had a look through and it seems that the acquisition of the `dataTypeAlias` doesnt seem to work. Im not 100% sure what `getDataTypeAlias()` is fully meant to do, but in my case all the values come through as empty strings, therefore not retrieving what is expected. I have however done a little re-work and got it to work with the following code (may need refining, but again, I dont know if `GetDataTypeAlias` is meant to be a shortcut to doing this):

```suggestion
		var dataTypeAlias = "";

		var dataTypeGuid = context.ContentTypes.GetEditorAliasByTypeAndProperty(contentProperty.ContentTypeAlias, contentProperty.PropertyAlias)?.DataTypeDefinition ?? Guid.Empty;

		if (dataTypeGuid != Guid.Empty)
        {
            dataTypeAlias = context.DataTypes.GetByDefinition(dataTypeGuid)?.DataTypeName;
        }
```

**Comment by ryan-woodruff** [comment]
I have also noticed that there are some changes missing from the original PR in `uSync.Migrations/Migrators/BlockGrid/Config/GridToBlockGridConfigLayoutBlockHelper.cs` which are vital to assigning the settings models to the grids

File: uSync.Migrations/Migrators/BlockGrid/GridToBlockGridMigrator.cs (hunk 1 above)
Comment (bielu) [reply uSync.Migrations/Migrators/BlockGrid/GridToBlockGridMigrator.cs:121]: Thanks for comments will check them tomorrow :)

**Comment by bielu** [comment]
Replaced by https://github.com/Jumoo/uSyncMigrations/pull/232

**Pull request closed without merging**